fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -3002,6 +3002,15 @@ public class Config extends ConfigBase {
     })
     public static int backup_restore_batch_task_num_per_rpc = 10000;
 
+    @ConfField(mutable = true, masterOnly = true, description = {
+            "一个 BE 同时执行的恢复任务的并发数",
+            "The number of concurrent restore tasks per be"})
+    public static int restore_task_concurrency_per_be = 5000;
+
+    @ConfField(mutable = true, description = {"BE心跳超过多长时间，认为BE不可用",
+            "The time after which BE is considered unavailable if the heartbeat is not received"})
+    public static int be_unavailable_heartbeat_timeout_second = 300;
```

Comment:
The config is used by agent task, so we'd better put agent task in config name, otherwise it is confused with hearbeat timeout.